podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+
+class LoginCoordinator: ObservableObject {
```

Comment:
```suggestion
class LoginCoordinator {
```

(maybe you're planning having states published here, if that's the case just ignore my comment)